.github/workflows/migrate.yml

```
@@ -0,0 +1,66 @@
+            echo "DATABASE_URL=${{ secrets.STAGING }}" > .env
+
+      - name: Backup Database before migration
+        run: |
+          sudo systemctl start postgresql.service
+          if [ "${{ github.event.inputs.environment }}" == "production" ]; then
+            pg_dump ${{ secrets.PRODUCTION }} > backup.sql
+          else
+            pg_dump ${{ secrets.STAGING }} > backup.sql
+
+      - name: Upload Database Backup
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)